### PR TITLE
fallback to OSS on license expiry

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: mirrord/operator/src/client.rs [hunk 1]
```diff
@@ -287,6 +285,13 @@ impl OperatorApi {
                 progress.info(&good_validity_message);
                 info!(good_validity_message);
             }
+        } else {
+            let no_license_message = "Operator has no valid license, falling back to OSS usage.";
```
Comment (eyalb181) [inline mirrord/operator/src/client.rs:289]:
```suggestion
            let no_license_message = "No valid license found for mirrord for Teams, falling back to OSS usage. Visit https://app.metalbear.co to purchase or renew your license.";
```

File: mirrord/operator/src/client.rs [hunk 2]
```diff
@@ -102,6 +102,9 @@ pub enum OperatorApiError {
         operation: String,
         status: Box<kube::core::Status>,
     },
+
+    #[error("Operator has expired license, falling back to OSS usage.")]
+    NoLicense,
```
Comment (meowjesty) [inline mirrord/operator/src/client.rs:107]:
```suggestion
    ExpiredLicense,
```

Nit.

Comment (meowjesty) [reply mirrord/operator/src/client.rs:107]: And if you go with this, change the `no_license` vars to `expired_license`.

**Review by meowjesty — APPROVED** [review]
Just a nit on the error name.

File: mirrord/operator/src/client.rs (hunk 2 above)
Comment (meowjesty) [reply mirrord/operator/src/client.rs:107]: Sure, it was only a nit anyway.

**Pull request merged**